app/components/user_form/user_form.html.erb

```diff
@@ -1,42 +1,46 @@
-<%= form_with(model: user, local: true, class: "UserForm", data: { controller: "user-form", action: "input->user-form#setDirty reset->user-form#resetDirty" } ) do %>
+<%= form_with(model: user, local: true, class: "UserForm" ) do %>
   <%= c 'stack' do %>
     <%= c 'field', object: user, id: :first_name, styles: [:horizontal] do |field| %>
-      <%= c 'input', field.input_defaults.merge(disabled: true) %>
```

Comment:
That actually isn’t directly related to this PR and just (partly) reverts a hot fix/workaround we added for our Prototype fund demo (where we wanted to prevent testers from adding/editing users).